localdev/Dockerfile

```diff
@@ -0,0 +1,33 @@
+## This Dockerfile is used for building Horust in a conatiner while developing locally.
+## It follows https://blog.mgattozzi.dev/caching-rust-docker-builds/ AND SHOULD NOT BE USED IN PRODUCTION
+
+FROM rust:1.42 as builder
+WORKDIR /usr/src/myapp
+ARG CARGO_PARAMS
+ARG GIT_COMMIT
```

Comment:
these are not needed for the "development" container

Reply:
Removed